ReportingServicesTools/Functions/CatalogItems/Rest/New-RsCacheRefreshPlan.ps1

```diff
@@ -0,0 +1,115 @@
+# Copyright (c) 2016 Microsoft Corporation. All Rights Reserved.
+# Licensed under the MIT License (MIT)
+
+function New-RsRestCacheRefreshPlan
+{
+    <#
+        .SYNOPSIS
+            This function creates a new CacheRefreshPlan for the specified Power BI Report
+
+        .DESCRIPTION
+            This function creates a new CacheRefreshPlan for the specified Power BI Report
+
+        .PARAMETER RsItem
+            Specify the location of the Power BI Report
+
+        .PARAMETER ReportPortalUri
+            Specify the Report Portal URL to your Power BI Report Server Instance.
+
+        .PARAMETER Recurrence
+            Specify the recurrence frequency
+
+        .PARAMETER RestApiVersion
+            Specify the version of REST Endpoint to use. Valid values are: "v2.0".
+
+        .PARAMETER Credential
+            Specify the credentials to use when connecting to the Report Server.
+
+        .PARAMETER WebSession
+            Specify the session to be used when making calls to REST Endpoint.
+
+        .EXAMPLE
+            New-RsRestCacheRefreshPlan -RsItem /ReportCatalog
+
+            Description
+            -----------
+            Creates a new CacheRefreshPlan for the ReportCatalog Power BI Report under "/" parent folder.
```

Comment:
I suggest to add a couple of examples with the recurrence parameter

Reply:
I have added two examples for the ` -Recurrence` parameter.